bite-size-notes/week-7-EKF.md

```diff
@@ -0,0 +1,184 @@
+ # Extended Kalman Filter
+
+## Scope and Objectives
+Since the normal Kalman filter can only deal with linear dynamics problems, extended Kalman filter(EKF) is the nonlinear version of the Kalman filter which linearizes about an estimate of the current mean and covariance. 
+
+In this note, I intend to provide a brief introduction to EKF, specifying EKF problems, and explaining its relevance to control theories. After reading this note, readers will understand the basic concepts and motivations with EKF.  
```

Comment:
Generally, notes are referred to in the plural sense, consider changing "reading this note" to "reading these notes"